mlir/python/CMakeLists.txt

```diff
@@ -30,6 +29,14 @@ declare_mlir_python_sources(MLIRPythonSources.Core.Python
     _mlir_libs/_mlir/passmanager.pyi
 )
 
+declare_mlir_python_sources(MLIRPythonSources.Core.Python.Extras
+  ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/mlir"
+  ADD_TO_PARENT MLIRPythonSources.Core.Python
+  SOURCES
+    extras/types.py
```

Comment:
Just so you are aware, I recently started colonizing this extras/ folder as well. See https://github.com/llvm/torch-mlir/blob/main/python/CMakeLists.txt#L38 . Just so we are aware this is now a shared namespace of sorts.

Reply:
Yea I've started "colonizing" in various extracurricular places as well. I think it should be fine - it's a `NameSpace` package after all so we can all share. But it's a good point to be mindful of whether downstream people have started populating (the produced effect is downstream overwrites which isn't fun...).